src/tiledb/cloud/invites.py

```diff
@@ -0,0 +1,165 @@
+from typing import List
+
+from tiledb.cloud import client
+from tiledb.cloud import rest_api
+from tiledb.cloud import tiledb_cloud_error
+from tiledb.cloud._common import utils
+from tiledb.cloud.rest_api.models.invitation_organization_join_email import (
+    InvitationOrganizationJoinEmail,
+)
+from tiledb.cloud.rest_api.models.organization_roles import OrganizationRoles
+
+
+def accept_invitation(invitation_id: str) -> None:
+    """
+    Accept an invitation.
+
+    :param str invitation_id: the ID of invitation about to be accepted.
```

Comment:
Since we have type annotations in the signature, there is no need to include them in the params, so this can just be `:param invitation_id:`